presto-kinesis/src/main/java/io/prestosql/plugin/kinesis/KinesisConfig.java

```diff
@@ -44,6 +44,7 @@
     private Duration checkpointInterval = new Duration(60000, TimeUnit.MILLISECONDS);
     private String logicalProcessName = "process1";
     private int iteratorNumber;
+    private int updateInterval;
```

Comment:
What would be the default value for this ?

Reply:
600 was the previous default. Let's keep that.